### DISCUSSION TITLE
Blogpost #8

### DISCUSSION THREAD

**Discussion by lucariebel** [lucariebel/online-shop#10] [General]
In der letzten Woche wurde ein Teil des [SAD](https://github.com/lucariebel/online-shop/blob/main/doc/SoftwareArchitectureDocument.md) erstellt und zusätzlich der User-Login mit Authentifizierung implementiert.

Abschnitt 8 wird morgen noch nachgereicht.

Vorläufig sieht das Formular zum anmelden/registrieren so aus:

![Screenshot 2024-12-02 164510](https://github.com/user-attachments/assets/145ac2f9-31d2-4f75-bb63-953f166ac94f)

![Screenshot 2024-12-02 164536](https://github.com/user-attachments/assets/16e80858-d5ff-4af4-8344-35b44978028d)

Viele Grüße
Euer WebShop24

**Comment by Yahya-EzEd** [comment]
Hallo Zusammen,

Das sieht alles sehr spannend aus Das Software Architecture Document ist sehr übersichtlich gestaltet.
Es kann sein, dass es bei mir nicht lädt aber in welchem Abschnitt sind die Komponentendiagramme zu finden?

LG 
Restaurant Reservierung

**Comment by Mimikry2** [comment]
Moin,

das Software Architecture Document bietet eine gut strukturierte und nachvollziehbare Übersicht über die Architektur der Plattform. Besonders gelungen ist die klare Unterteilung in logische, prozessuale und physische Sichten. Die Nutzung moderner Technologien wie Angular, ASP.NET, SQLite und Redis zeigt, dass aktuelle Best Practices der Softwareentwicklung berücksichtigt wurden. Die Beschreibung der Use-Case-Realizations, insbesondere die Sequenzdiagramme, erleichtert das Verständnis der Interaktionen und Abläufe im System.

Viele Grüße,
Tim, Dennis, Nico

**Comment by GermanJesus-lul** [comment]
Hey,

Das SAD sieht schon gut aus. 
Beim Login Design ist mir nur ins Auge gestochen, dass der Strich und der anschließende Button mir nicht so gefallen:  
![image](https://github.com/user-attachments/assets/c2c951f4-4c64-4317-8c6d-942e1e84d337)

Mit freundlichen Grüßen,
Julius und das Rezeptbuch Team
